packages/core/src/oidc/init.ts

```diff
@@ -309,7 +310,17 @@ export default function initOidc(
    *
    * Other parsers are explicitly disabled to keep it neat.
    */
-  oidc.use(koaBody({ urlencoded: false, text: false }));
+  oidc.use(async (ctx, next) => {
+    try {
+      await koaBody({ urlencoded: false, text: false })(ctx, next);
+    } catch (error: unknown) {
+      if (error instanceof SyntaxError) {
```

Comment:
better have a comment to describe why it should be specially handled

Reply:
updated